docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -1250,9 +1248,28 @@ pantheon password hash --password=<my-password>
 pantheon password hash --password=myPassword123
 ```
 
+### operator
+
+Provides password related actions.
+
+### generate-blockchain-config
+
+This command generates [IBFT 2.0 configuration files](../Tutorials/Create-IBFT-Network.md). 
+
+```bash tab="Syntax"
+pantheon operator generate-blockchain-config --config-file=<FILE> --to=<DIRECTORY> [--genesis-file-name=<PATH>] [--private-key-file-name=<PATH>] [--public-key-file-name=<PATH>]
+```
+
+```bash tab="Example"
+pantheon operator generate-blockchain-config --config-file=config.json --to=myNetworkFiles
+```
+
+The configuration file is an IBFT 2.0 genesis file except instead of the extraData property, the number of key pairs to 
```

Comment:
Not exactly the same structure. This config file has 2 JSON nodes at the rool level:
```javascript
{
    "genesis": {
       ...... classical genesis structure inside
    }
    "blockchain": {
      ....... specific config of the tool
    }

}
```

Reply:
Fixed